pages/hooks/usereducer.mdx

```diff
@@ -2,6 +2,78 @@
 title: useReducer
 ---
 
+import useReducer from '../../examples/files/hooks/useReducer.js'
+import useReducerRefactor from '../../examples/files/hooks/useReducerRefactor.js'
+import useReducerLazyInit from '../../examples/files/hooks/useReducerLazyInit.js'
+
 # useReducer
 
-Coming soon!
+useReducer is similar to useState since it allows us to control state inside a function component.  
+The difference between the two is that useReducer allows us to handle multiple states and control complex state changes. 
+
+The useReducer hook takes two mandatory arguments: a reducer function and an initial argument. It can also take in an optional third argument called the initializer, which we will discuss later. useReducer returns the current state and a dispatch function.  
+
+`const [state, dispatch] = useReducer(reducer, initialState)`
+
+## Reducer
+
+The reducer function is a pure function that takes in the previous state and an action. It returns a new state. 
```

Comment:
I would mention that `action` is an arbitrary value we choose that can be any type